lua/astrocommunity/project/linear-nvim/init.lua

```diff
@@ -8,7 +8,7 @@ return {
       "AstroNvim/astrocore",
       opts = function(_, opts)
         local maps = opts.mappings
-        local prefix = "<Leader>m"
+        local prefix = "<Leader>L"
 
         maps.n[prefix] = { desc = "Linear" }
         maps.n[prefix .. "m"] = {
```

Comment:
```suggestion
        maps.n[prefix .. "l"] = {
```

Reply:
I think the mapping descriptions for the other bindings also should be updated like "create issue" and "show issue"

Reply:
Why? Isnt the current mapping descriptions(based  on the original desc from the plugin readme) good enough?